corehq/apps/users/templates/users/email/user_limit_notice.html

```diff
@@ -0,0 +1,15 @@
+{% if at_capacity %}
```

Comment:
Would it be helpful to include the actual numbers vs capacity here? Specifically for the 90% message, I feel it would be useful to know that they could create another 5 or 500 users, for example

Reply:
I think you can take out the entire if statement here by using something along the lines of
```
"You have reached {created_users} out of {maximum_users} allowed in your plan. Your user limit represents all mobile users that are not currently deactivated
    (see <a href="{{ url }}">here</a> for more). If you exceed the number of mobile users
    included in your plan, you will be charged as per the terms of plan."
```

Reply:
The text is lifted directly from the specs and I'm not sure if I'm allowed to mess with it without approval.

Reply:
Yeah that makes sense. Would it make sense to speak to the creator of the spec about at least including actual numbers here instead?